client/js/widgets/card.js

```diff
@@ -149,7 +149,7 @@ class Card extends Widget {
 
   getDefaultValue(property) {
     if(this.deck) {
-      const d = this.deck.cardPropertyGet(this.p('cardType'), property);
+      const d = this.deck.cardPropertyGet(this.p('cardType'), this.p('id'), this.state.activeFace, property);
```

Comment:
`this.state` should usually not be used because it is _only exactly_ the widget state as it is in the JSON. If `activeFace` comes from anywhere else, this would not be correct. Please use `this.p('activeFace')` unless you have a good reason.

Reply:
I cannot find a way to use this.p('activeFace') in this method - I get a recursion error.